scripts/merge-fork/merge-fork.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+/* eslint-disable no-for-of-loops/no-for-of-loops */
+
+const {writeFileSync} = require('fs');
+const path = require('path');
+const {spawnSync} = require('child_process');
+const minimist = require('minimist');
+const tmp = require('tmp');
+
+const argv = minimist(process.argv.slice(2), {
+  boolean: ['reverse'],
+  default: {
+    'base-ref': 'HEAD',
+  },
+});
+
+const baseRev = argv['base-ref'];
+const baseDir = argv['base-dir'] || '';
```

Comment:
tiny nit: Why use `minimist`'s default mechanism for `base-ref` but inline default for `base-dir`? :smile:

Reply:
Idk! Good catch